service.upnext/resources/lib/utils.py

```diff
@@ -0,0 +1,166 @@
+import xbmc
+import xbmcgui
+import xbmcaddon
+import inspect
+import binascii
+import json
+
+
+def addon_id():
+    return "service.upnext"
+
+
+def addon_path():
```

Comment:
you don't need to specify the id: `xbmcaddon.Addon()` is enough. If `addon_id()` is unused after the refactoring please remove.